app/models/institution.rb

```diff
@@ -159,6 +159,8 @@ class Institution < ActiveRecord::Base
   validates :institution_type_name, inclusion: { in: TYPES }
 
   has_many :yellow_ribbon_programs, dependent: :destroy
+  has_many :school_certifying_officials, -> { order 'priority, last_name' },
+           primary_key: :facility_code, foreign_key: 'facility_code', dependent: :destroy
```

Comment:
Are you sure you want `dependent: :destroy`?
We usually have multiple versions of a given `institution`.

Say you have `institution(facility_code: 0001, version: 1)` corresponding to `school_certifying_official(facilty_code: 0001)`

And you have `institution(facility_code: 0001, version: 2)` corresponding to the same `school_certifying_official(facilty_code: 0001)`

With this configuration, if you delete `institution(facility_code: 0001, version: 1)`, then `school_certifying_official(facilty_code: 0001)` will be deleted and thus no longer available for `institution(facility_code: 0001, version: 2)`

Reply:
@jholton I appreciate the feedback, I will go ahead and make that change